### ISSUE TITLE
Set lock screen rotation direction for nexus7 wifi(flo)

### ISSUE THREAD

**Issue by xqqy** [ubports/ubuntu-touch#945]
### **Description of the bug**
Illustrations

    Device: Asus Nexus7 wifi(flo)
    Channel: 16.04/devel

For my nexus7, seems auto screen rotation direction has going down. But here is a problem:
the lock screen and the boot screen always use landscape and Scopes and apps use portrait.
Can I change this in setting? Or it just my devices's fault?

**Comment by doniks** [comment]
> the lock screen and the boot screen always use landscape 

that's "normal". By that I mean, it has been like this forever. boot screen has a fixed orientation. I think (most) phones are fixed to portrait, N7 is landscape

> and Scopes and apps use portrait.
> Can I change this in setting? Or it just my devices's fault?

could you be experiencing this: https://www.ifixit.com/Answers/View/255621/Automatic+screen+rotation+not+working 

tl;dr

turn it off. lay it on a flat surface. press on the back

**Comment by xqqy** [comment]
But what about  lock screen.
After experiencing, it works.
But lock screen are always fixed.
![img_20181031_065958](https://user-images.githubusercontent.com/28598660/47756249-51e9e900-dcdc-11e8-95db-1668a117bf63.jpg)

**Comment by doniks** [comment]
> After experiencing, it works.

great

> But lock screen are always fixed.

yes. as I said. Also see here https://github.com/ubports/ubuntu-touch/issues/275

**Comment by xqqy** [comment]
Thanks.

**Comment by Elowan** [comment]
hi there - so there is no way to fix this? I´d really like to have my "lockscreen" rotated "Portrait" and not landscape mode...
